website/docs/plugins/rq_launcher.md

```diff
@@ -0,0 +1,95 @@
+
+```yaml
+# @package hydra.launcher
+cls: hydra_plugins.hydra_rq_launcher.rq_launcher.RQLauncher
+params:
+  enqueue:
+    job_timeout: null                  # maximum runtime of the job before it's interrupted and marked as failed (in sec)
+    ttl: null                          # maximum queued time before the job before is discarded (in sec)
+    result_ttl: 864000                 # how long successful jobs and their results are kept (in sec), default: 10 days
+    failure_ttl: 8640000               # specifies how long failed jobs are kept (in sec), default: 100 days
+    at_front: false                    # place job at the front of the queue, instead of the back
+    job_id: null                       # job id, will be overidden automatically by a uuid unless specified explicitly
+    description: null                  # description, will be overidden automatically unless specified explicitly
+  queue: default                       # queue name
+  redis:
+    host: ${env:REDIS_HOST,localhost}  # host address via REDIS_HOST environment variable, default: localhost
+    port: ${env:REDIS_PORT,6379}       # port via REDIS_PORT environment variable, default: 6379
+    db: ${env:REDIS_DB,0}              # database via REDIS_DB environment variable, default: 0
+    password: ${env:REDIS_PASSWORD,}   # password via REDIS_PASSWORD environment variable, default: no password
+    mock: ${env:REDIS_MOCK,False}      # switch to run without redis server in single thread, for testing purposes only
+  stop_after_enqueue: false            # stop after enqueueing by raising custom exception
+  wait_polling: 1.0                    # wait time in seconds when polling results
+```
+
+The plugin is using environment variables to store Redis connection information. The environment variables `REDIS_HOST`, `REDIS_PORT`, `REDIS_DB`, and `REDIS_PASSWORD`, are used for the host address, port, database, and password of the server, respectively. 
+
+For example, they might be set as follows when using `bash` or `zsh` as a shell:
+
+```commandline
+export REDIS_HOST="localhost"
+export REDIS_PORT="6379"
+export REDIS_DB="0"
+export REDIS_PASSWORD=""
+```
+
+Assuming configured environment variables, workers connecting to the Redis server can be launched using:
+
+```commandline
+$ rq worker --url redis://:$REDIS_PASSWORD@$REDIS_HOST:$REDIS_PORT/$REDIS_DB
+```
+
+An [example application](https://github.com/facebookresearch/hydra/tree/master/plugins/hydra_rq_launcher/example) using this launcher is provided in the plugin repository.
+
+Starting the app with `python my_app.py --multirun task=1,2,3,4,5` will enqueue five jobs to be processed by worker instances:
+
+```text
+$ python my_app.py --multirun task=1,2,3,4,5
+
+[HYDRA] RQ Launcher is enqueuing 5 job(s) in queue : default
+[HYDRA] Sweep output dir : multirun/2020-06-15/18-00-00
+[HYDRA] 	#1 : task=1
+[HYDRA] Enqueued 13b3da4e-03f7-4d16-9ca8-cfb3c48afeae
```

Comment:
Those prints can be cleared.
maybe reverse their order. (even if it means moving the params print to after the job is submitted).